client/state/document-head/selectors.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { compact } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { decodeEntities } from 'lib/formatting';
+import { getSelectedSiteId, isSiteSection } from 'state/ui/selectors';
+import { getSiteTitle } from 'state/sites/selectors';
+
+/**
+ * Returns the document title as set by the DocumentHead component or setTitle
+ * action.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {?String}        Document title
+ */
+export function getTitle( state ) {
+	return state.documentHead.title;
+}
+
+/**
+ * Returns a count reflecting unread items.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {?String}        Unread count (string because it can be e.g. '40+')
+ */
+export function getUnreadCount( state ) {
+	return state.documentHead.unreadCount || 0;
```

Comment:
The default value for the `unreadCount` reducer is zero, so do we need this fallback?